src/app/zap-templates/zcl/data-model/chip/power-source-cluster.xml

```diff
@@ -63,7 +63,6 @@ limitations under the License.
     <attribute side="server" code="0x001D" define="POWER_SOURCE_BAT_CHARGING_CURRENT" type="INT32U" min="0x00000000" max="0xFFFFFFFF" writable="false" optional="true">BatteryChargingCurrent</attribute>
     <attribute side="server" code="0x001E" define="POWER_SOURCE_ACTIVE_BAT_CHARGE_FAULTS" type="ARRAY" entryType="ENUM8" length="16" writable="false" optional="true">ActiveBatteryChargeFaults</attribute>
 
-    <!-- TODO: Do events work?
     <event code="0x0000" name="WiredFaultChange" priority="info" side="server">
```

Comment:
Where is the priority for these defined?  The spec definition of these (and in fact the whole cluster in the spec) does not follow the normal cluster template, so it's hard to tell whether these are correct.